### PR TITLE
Fix DEFAULT_BRANCH variable substitution

### PR DESCRIPTION
**Pull request by acchaulk**
Check DEFAULT_BRANCH against CI_COMMIT_REF_NAME since variable substitution does not work in 'only'

### CODE REVIEW

**Comment by acchaulk** [comment]
Tested this out. It appears that variable substitution doesn't work with [only](https://docs.gitlab.com/ee/ci/variables/where_variables_can_be_used.html). The pipeline wasn't triggered on master, but it was when I created a branch named "$DEFAULT_BRANCH"

File: belt-node8-service.gitlab-ci.yml [hunk 1]
```diff
@@ -47,7 +47,8 @@ publish:
   script:
     - grunt bundle --packageVersion=$VERSION --packageName=$CI_PROJECT_NAME
   only:
-    - $DEFAULT_BRANCH
+    variables:
+      - $CI_COMMIT_REF_NAME == $DEFAULT_BRANCH
```
Comment (acchaulk) [inline belt-node8-service.gitlab-ci.yml:51]: alternatively, we could change this to check if the branch name is master OR develop

Comment (zmarois) [reply belt-node8-service.gitlab-ci.yml:51]: Have we tested out what overriding each job would look like? Or yaml inheritance, and overriding just the root?

Wish we'd just standardize our default branch

**Review by zmarois — DISMISSED** [review]

File: belt-node8-service.gitlab-ci.yml (hunk 1 above)
Comment (zmarois) [reply belt-node8-service.gitlab-ci.yml:51]: Or we just have our own

Comment (briangweber) [reply belt-node8-service.gitlab-ci.yml:51]: I could just go through and swap our stuff to master. Seems like we've already spent too much time on this.

**Pull request closed without merging**